library/include/borealis/core/application.hpp

```diff
@@ -204,6 +205,17 @@ class Application
      * Returns the current system locale.
      */
     static std::string getLocale();
+    
+    /**
+     * If touch input mode enabled, disable it and move focus on last view
+     * Returns true if touch mode was disabled.
+     */
+    static bool dismissTouchMode();
```

Comment:
Actually, for the sake of extensibility, I wonder if it would not be better to use an enum here instead of a boolean. I'm thinking of an eventual "mouse mode" that would work yet again differently from both touch and controller mode.